src/Monaco/customTokenThemeRules.js

```diff
@@ -0,0 +1,6 @@
+export const customTokenThemeRules = [
+    {token: 'custom-negation.gitignore', foreground: 'c00ce0'},
+    {token: 'string.bold', fontStyle: 'bold'},
+    {token: 'string.emphasis', fontStyle: 'italic'},
+    {token: 'string.underline', fontStyle: 'underline'}
```

Comment:
Don't we need combined styles for the nested formats? Or is this to complicated?